docs/tutorial/part-one/index.md

```diff
@@ -1,390 +1,282 @@
 ---
-title: Introduction to Gatsby basics
+title: Familiarize with Gatsby Building Blocks
 typora-copy-images-to: ./
 ---
 
-Hello fellow Gatsby-er! Welcome to _part one_ of our community Gatsby.js
-tutorial.
+In the [previous section](/tutorial/part-zero/), we got our development environment all ready to go by installing different software and scaffolding out our first Gatsby site using the [“hello world” starter](https://github.com/gatsbyjs/gatsby-starter-hello-world). Now we’re going to take a deeper dive into what exactly the [Gatsby](https://github.com/gatsbyjs/gatsby-starter-default)’s [default starter](https://github.com/gatsbyjs/gatsby-starter-default) gives us.
 
-## What's in this tutorial?
+## Gatsby’s default starter
 
-In this tutorial you'll be gently introduced to the Gatsby development
-environment, how to create component pages, and how to build and deploy Gatsby
-sites.
+In the previous section, we created a site based on the “hello world” starter using the following command:
 
-Sit down, buckle up, and let's get going!
+`gatsby new hello-world https://github.com/gatsbyjs/gatsby-starter-hello-world`
 
-## Check your development environment
+When creating a new Gatsby site, you can use the following command structure to create a new site based on any existing Gatsby starter:
 
-Let's check first that you have everything set up to start creating with Gatsby.
-You will need a recent version of Node.js installed.
+`gatsby new [SITE_DIRECTORY] [URL_OF_STARTER_GITHUB_REPO]`
 
-Node.js is a programming tool for running JavaScript on servers and in your
-computer's terminal. Gatsby is built using Node.js.
+> 💡 See a list of the existing [official and community starters](/docs/gatsby-starters/)!
 
-Open a terminal window. See
-[terminal instructions for Mac users](http://www.macworld.co.uk/feature/mac-software/how-use-terminal-on-mac-3608274/) and
-[terminal instructions for Windows users](https://www.quora.com/How-do-I-open-terminal-in-windows). In your terminal window, type `node --version` and hit ENTER, then `npm --version` and hit ENTER (tip: to run a specified command, you must type the command into your terminal and then press ENTER. Then the command will run).
+If you omit a URL from the end, Gatsby will automatically generate a site for you based on the [default starter](https://github.com/gatsbyjs/gatsby-starter-default). Let’s do that now, to create a new site with the default starter, and explore the output.
 
-You should see something like:
+### ✋ Generate a new site based on Gatsby’s default starter
 
-![Check if node.js/npm is installed](check-versions.png)
+1. Open up your terminal. 
+2. Run `gatsby new my-tutorial-site`.
+3. Run `cd my-tutorial-site`.
+4. Run `gatsby develop`.
 
```

Comment:
I ended up needing to run `npm install` before gatsby develop. Maybe I didn't have it installed globally?

Reply:
This is dir/project specific, not global -- and is consistent with the quick start docs. https://www.gatsbyjs.org/docs/ 

I just did it again for sanity check and had no issue. Scaffolding out a new site using `gatsby new` does the install for you. Can you follow all four steps again exactly, and if it happens again send me the output, so I can add messaging for whatever is happening for you? (If it's happening for you, or it's a misunderstanding, it's likely to be shared by others).

Reply:
Ok, after discussing this with some co-workers, there are some arguments to use the hello-world starter instead. I apologize for not catching that earlier! Let me make sure I get a thorough understanding of their reasoning @KyleAMathews?